derailed/app.py

```diff
@@ -1,83 +1,56 @@
-import dotenv
-from gevent import monkey
+"""
```

Comment:
do we really need to do the "add license on every file thing"? it's a lot of lines for 0 benefit

Reply:
I'm a person very conscious of anyone stealing my code or any work I've done. And that makes it the case that I normally always put licenses on anything I can so I can get credit and if code is stolen, it's blatant and not unknown.

Reply:
if I wanted to steal your code I wouldn't leave the license in yk